### PR TITLE
Update to use libswoc 1.3.0 (Errata changes).

### PR DESCRIPTION
**Pull request by SolidWallOfCode**

### CODE REVIEW

File: plugin/src/0_static.cc [hunk 1]
```diff
@@ -15,3 +21,11 @@ Extractor::Table Extractor::_ex_table;
...
 /// Static mapping from modifier to factory.
 Modifier::Factory Modifier::_factory;
+
+const bool TXN_BOX_LIB_INIT = []() -> bool {
+  std::array<swoc::TextView, 5> S_NAMES = { "Success", "Debug", "Info", "Warning", "Error"};
+  swoc::Errata::SEVERITY_NAMES = swoc::MemSpan<swoc::TextView>{S_NAMES.data(), S_NAMES.size()};
+  swoc::Errata::DEFAULT_SEVERITY = S_ERROR;
+  swoc::Errata::FAILURE_SEVERITY = S_ERROR;
+  return true;
+}();
```
Comment (brbzull0) [inline plugin/src/0_static.cc:31]: This is mandatory unless we use the defaults right?

File: plugin/include/txn_box/common.h [hunk 2]
```diff
@@ -32,21 +37,6 @@ namespace YAML
...
-  return std::move(swoc::Errata().note_v(swoc::Severity::ERROR, fmt, std::forward_as_tuple(args...)));
-}
-
-template <typename... Args>
-swoc::Errata
-Warning(std::string_view const &fmt, Args &&... args)
-{
-  return std::move(swoc::Errata().note_v(swoc::Severity::WARN, fmt, std::forward_as_tuple(args...)));
-}
-
```
Comment (brbzull0) [inline plugin/include/txn_box/common.h:49]: Why removing it? I think this was a nice way of expressing the error/warn,etc
```
template <typename... Args>
swoc::Errata
Error(std::string_view const &fmt, Args &&... args)
{
  return swoc::Errata(S_ERROR, fmt, std::forward_as_tuple(args...)));
}
```

**Review by brbzull0 — APPROVED** [review]
looks good! just 2 notes

File: plugin/include/txn_box/common.h (hunk 2 above)
Comment (SolidWallOfCode) [reply plugin/include/txn_box/common.h:49]: It's just a severity code to be passed in now, I didn't think it was worth it.

**Pull request merged**